framework/include/base/MooseApp.h

```diff
@@ -66,6 +67,14 @@ class MooseApp : public ConsoleStreamInterface,
                  public libMesh::ParallelObject
 {
 public:
+  struct SolveConfig
```

Comment:
Change `SolveConfig`  to `FixedPointSolveConfig` since it is unrelated to nonlinear solvers